### DISCUSSION TITLE
Welcome to artistd Discussions!

### DISCUSSION THREAD

**Discussion by TwinkieXLII** [JGMaker3dofficial/artistd#21] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dero99** [comment]
Greetings!  I’m Deró and I have built three 3D Printers and I have supported some Kickstarter projects as well. Unfortunately the success rate of those is poor and I’m still trying to be positive about this one. 
Still waiting for the second North American shipment to arrive with my device.

**Reply by mikejr83** [reply]
This is my second Kickstarter printer. The previous one I backed was a disaster. The MKS Gen L v2 board was dead on arrival. The motion system in it sucked so bad that another backer and myself completely redesigned it. I added in linear rails and bunch of other stuff. It's now a pile of sheet metal under my workbench. The electronics are waiting for me to have the courage to order a frame kit from Ali or Banggood. The company didn't do much if anything to address any concerns. 
So far I think this printer is pretty good. It took a bit more effort to get it going than my Ender 3. However, when I look at the build quality and what is sitting on my desk I can't say that I could be more happier with what I got. My biggest hope in the future is that some of the guys that are doing hardware hacks will trickle over into supported parts by JGMaker. 
Software-wise, we're working on it. I think there were some stumbling blocks a little early. It would have been really nice if there was a repository all ready in place. I think we could have made some suggestions and fixes that got out to everyone before the printer shipped. That said, I think since Marlin is an open source project and @attacktwinkie and JGMaker have laid the ground work for collaboration we'll see some really cool stuff come out. With as many backers getting this printer we're going to be pushing some boundaries in Marlin with respect to IDEX. We have an opportunity to make not only our firmware great for this printer but make a positive mark on other printers as well. The same goes for some of the other IDEX printers that showed up right when this campaign started. I don't have as much info on the Wedo (sp?) but it is a similar IDEX machine and we may be able to find some enhancements that we can employ here.
All in all, I'm excited!

**Reply by Dero99** [reply]
I did back Weedo as well, but dropped out at last min. The strengths of Weedo are weakness in ArtistD and vis versa. But main reason I dropped Weedo was proprietary controller and closed sourced.

**Reply by TwinkieXLII** [reply]
That would be a good reason to drop them.

**Reply by mikejr83** [reply]
Didn't know that Weedo wasn't open source. Ya, that would have kept me from backing. It's not a closed vs. open thing, but a company that I've never heard of offering a complex IDEX printer on Kickstarter doesn't inspire that much confidence from me. I look at my graveyard of consumer electronics that I backed on KS and are essentially dead because the company went under and shake my head. So much nice hardware sitting here but since it's closed it's basically worthless.

**Comment by genekimble** [comment]
Do you have a new firmware that will allow printing higher than 260?
